SimGeneral/MixingModule/plugins/MixingModule.h

```diff
@@ -56,6 +57,8 @@ namespace edm {
     /**Default destructor*/
     ~MixingModule() override;
 
+    static void fillPSetDescription(edm::ParameterSetDescription& iDesc);
```

Comment:
MixingModule is a framework module, so to have any effect the function should be
```suggestion
    static void fillDescriptions(edm::ConfigurationDescriptions& iDesc);
```

Reply:
thanks @makortel for the comments, when I do this I get
```
error: 'class edm::ConfigurationDescriptions' has no member named 'setAllowAnything'
```

Reply:
ok I guess I made it work, I'll push in a second